amd_openvx_extensions/amd_rpp/source/image/SequenceRearrangebatchPD.cpp

```diff
@@ -0,0 +1,294 @@
+/*
+Copyright (c) 2019 - 2023 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include "internal_publishKernels.h"
+
+struct SequenceRearrangebatchPDLocalData
+{
+    vxRppHandle *handle;
+    RppiSize dimensions;
+    RppPtr_t pSrc;
+    RppPtr_t pDst;
+    Rpp32u device_type;
+    vx_uint32 new_sequence_length;
+    vx_uint32 sequence_length;
+    vx_uint32 sequence_count;
+    vx_uint32 *new_order;
+#if ENABLE_OPENCL
+    cl_mem cl_pSrc;
```

Comment:
I think I have commented on this before. We shouldn't mix camel and snake_case both. Please keep everything in single type

Reply:
This is a batchPD file. In our previous PR we had modified the batchPD code itself. So we have have reintroduced the file and changed name alone. In all our tensor files we have followed the naming convention in all files